src/main/resources/application.conf

```diff
@@ -38,110 +46,221 @@ etc-client {
         "enode://fba5a07e283d517a2680bcfc7aeb498ac2d246d756556a2ebd5edeb39496491c47a6d27e27f82833b7d7d12defc8de994de04bb58beb72472649f9a323006820@41.135.121.6:30303"
       ]
 
+      # TODO: remove me
       bootstrap-nodes-scan-interval = 2 minutes
     }
 
     peer {
+      # Retry delay for failed attempt at connecting to a peer
       connect-retry-delay = 20 seconds
+
+      # Maximum number of reconnect attempts after the connection has been initiated.
+      # After that, the connection will be dropped until its initiated again (eg. by peer discovery)
       connect-max-retries = 30
+
       disconnect-poison-pill-timeout = 5 seconds
+
       wait-for-hello-timeout = 3 seconds
+
       wait-for-status-timeout = 30 seconds
+
       wait-for-chain-check-timeout = 15 seconds
 
+      # Maximum block headers in a single response message (as a blockchain host)
       max-blocks-headers-per-message = 200
+
+      # Maximum block bodies in a single response message (as a blockchain host)
       max-blocks-bodies-per-message = 200
+
+      # Maximum transactions receipts in a single response message (as a blockchain host)
       max-receipts-per-message = 200
+
+      # Maximum MPT components in a single response message (as a blockchain host)
       max-mpt-components-per-message = 400
 
+      # Maximum connected peers
       max-peers = 10
+
+      # Ethereum network identifier:
+      # 1 - mainnet, 2 - morden
       network-id = 1
     }
 
     rpc {
+      # Whether to enable JSON-RPC HTTP endpoint
       enabled = true
+
+      # Listening address of JSON-RPC HTTP endpoint
       interface = "127.0.0.1"
+
+      # Listening port of JSON-RPC HTTP endpoint
       port = 8546
+
+      # Enabled JSON-RPC APIs over the HTTP endpoint
+      # Available choices are: eth, web3, net, personal
       apis = "eth,web3,net"
     }
   }
 
   txPool {
+    # Maximum number of pending transaction kept in the pool
     tx-pool-size = 1000
+
     pending-tx-manager-query-timeout = 5.seconds
   }
 
   mining {
+    # Maximum number of ommers kept in the pool
     ommers-pool-size = 30
+
+    # Block cache size
+    # TODO: explain better, what is this cache?
```

Comment:
this is size of cash that stores blocks requested in getWork rpc methode, this cache is necessary to be able to get them when miner will submit work with different rpc methode